app/client/components/delivery/records/deliveryRecords.tsx

```diff
@@ -103,11 +107,12 @@ export const checkForExistingDeliveryProblem = (
   records: DeliveryRecordDetail[]
 ) =>
   records.findIndex(deliveryRecord => {
-    const recordDateEpoch = moment(deliveryRecord.deliveryDate).unix();
-    const fourteenDaysAgoEpoch = moment()
-      .startOf("day")
-      .subtract(14, "days")
-      .unix();
+    const recordDateEpoch = parseDate(
+      deliveryRecord.deliveryDate
+    ).date.valueOf();
+    const now = new Date();
```

Comment:
It looks like we only use `now` in one place. Might we get rid of it as a variable to reduce cognitive burden?